### PR TITLE
Some Redundancy Code Cleanup

### PR DESCRIPTION
**Pull request by HeavenTonight**
Signed-off-by: guiyong.ou <guiyong.ou@daocloud.io>

### Proposed changelog entries

Some Redundancy Code Cleanup

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")` if applicable.
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/6737"></a>

### CODE REVIEW

**Review by basil — APPROVED** [review]
Thanks for the PR!

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

File: core/src/main/java/jenkins/org/apache/commons/validator/routines/UrlValidator.java
Comment (daniel-beck) [inline core/src/main/java/jenkins/org/apache/commons/validator/routines/UrlValidator.java:343]: https://github.com/jenkinsci/jenkins/blob/c53b5bac8da3764973111f153e9eb1e6ea3b6112/core/src/main/java/jenkins/org/apache/commons/validator/routines/UrlValidator.java#L17 makes it clear that changes from upstream should be marked as such to make it easier to keep this in sync.

That said, this doesn't seem relevant enough to justify deviating from upstream.

**Review by daniel-beck — CHANGES_REQUESTED** [review]
The change to `UrlValidator` should be reverted.

**Review by basil — APPROVED** [review]

File: core/src/main/java/jenkins/org/apache/commons/validator/routines/UrlValidator.java
Comment (HeavenTonight) [reply core/src/main/java/jenkins/org/apache/commons/validator/routines/UrlValidator.java]: Thanks for reminding,i ignored it

**Review by daniel-beck — APPROVED** [review]

**Pull request merged**
